docs/source/alfacase_definitions/BipDescription.txt

```diff
@@ -7,12 +7,13 @@
         class BipDescription
             component_1: str
             component_2: str
-            value: float
+            value: float | \ :class:`FloatExpression <alfasim_sdk._internal.alfacase.case_description_attributes.FloatExpression>`
```

Comment:
Update the generator do do


```suggestion
            value: float | \ :class:`FloatExpression <alfasim_sdk.FloatExpression>`
```

Probably we want `FloatExpression` to be a public entity so ppl can programmatically add them to their `CaseDescription`s.